### ISSUE TITLE
Provide Accessibility

### ISSUE THREAD

**Issue by dovematrix** [google/shaka-player#552]
The WAVE toolbar for Chrome reports two accessibility errors - missing form labels on the scrub bar and volume control.

**Comment by joeyparrish** [comment]
Thank you for letting us know!  Accessibility is important, but we have not had any rigorous testing around that.

Is this the extension you are using?  http://wave.webaim.org/extension/  I want to make sure we can reproduce your results correctly.

**Comment by dovematrix** [comment]
Thanks, yes that is correct.
